eng/helix/content/RunTests/RunTests.csproj

```diff
@@ -1,13 +1,21 @@
-<Project Sdk="Microsoft.NET.Sdk">
-
+<Project>
   <PropertyGroup>
     <OutputType>Exe</OutputType>
     <TargetFramework>net7.0</TargetFramework>
     <DefineConstants Condition=" '$(InstallPlaywright)' == 'true' ">$(DefineConstants);INSTALLPLAYWRIGHT</DefineConstants>
+    <NoWarn>$(NoWarn);CA2007;NU5104</NoWarn>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <IsUnitTestProject Condition=" '$(SkipTestBuild)' != 'true' ">true</IsUnitTestProject>
```

Comment:
This is the opposite of what you want. `$(IsUnitTestProject)` defaults to `true` and that's causing Arcade to attempt to run tests in this project.

nit: Only this `<PropertyGroup/>` needs to be above the Sdk.props `<Import/>`

Reply:
> opposite of what you want

```
<IsUnitTestProject Condition=" '$(SkipTestBuild)' != 'true' ">false</IsUnitTestProject>
```

is what's intended?

Reply:
I wouldn't bother w/ the `Condition` but that's not a big deal at all. Otherwise fine.

Reply:
[576767a](https://github.com/dotnet/aspnetcore/pull/41311/commits/576767a7c5ca025db8d629ec47c7d5874eba566c)

Reply:
Still the same thing?

> eng/helix/content/RunTests/RunTests.csproj(0,0): error MSB4057: (NETCORE_ENGINEERING_TELEMETRY=Build) The target "Test" does not exist in the project.

Reply:
Opened a binary log from this last build and see `Target "Test" skipped, due to false condition; ('$(IsUnitTestProject)' == 'true' or '$(IsPerformanceTestProject)' == 'true') was evaluated as ('false' == 'true' or 'false' == 'true').` for adjacent projects. That indicates the `Test` target is successfully defined by Arcade's Tests.targets file elsewhere. 

OIC 🤦: eng/helix/ contains Directory.Build.* files that prevent `dotnet msbuild` seeing our root Directory.Build.* files, meaning the Arcade SDK doesn't get imported. Simple fix is to revert everything related to `$(IsTestProject)`, `$(IsUnitTestProject)`, go back to the regular .NET SDK approach, and add `<Target Name="Test" />` to the project.

Sorry again for my unhelpfulness here. Few too many things going on I guess&hellip;